tests/e2e/specs/auth-flow.test.js

```diff
@@ -34,6 +39,7 @@ describe( 'Site Kit set up flow for the first time', () => {
 
 	beforeAll( async() => {
 		await activatePlugin( 'e2e-tests-oauth-callback-plugin' );
+		await setSearchConsoleProperty();
```

Comment:
As mentioned, this is a general prerequisite for a ton of tests (now and future), so it should be a utility plugin I believe. Setting it _during_ a test should only happen in the specific test for Search Console setup.